lib/src/card/Card.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import React from "react";
+import Title from "../../.storybook/components/Title";
+import ExampleContainer from "../../.storybook/components/ExampleContainer";
+import DxcCard from "./Card";
+import imagePath from "./ice-cream.jpg";
+import { userEvent, within, fireEvent } from "@storybook/testing-library";
+
+export default {
+  title: "Card",
+  component: DxcCard,
+};
+
+const Card = () => (
+  <>
+    <Title title="Default" theme="light" level={4} />
+    <ExampleContainer>
+      <DxcCard>Default</DxcCard>
+    </ExampleContainer>
+    <Title title="Outlined" theme="light" level={4} />
+    <ExampleContainer>
+      <DxcCard outlined>Outlined</DxcCard>
+    </ExampleContainer>
+    <Title title="Default with link" theme="light" level={4} />
+    <ExampleContainer>
+      <DxcCard linkHref="https://www.dxc.com">Default with link</DxcCard>
+    </ExampleContainer>
+    <Title title="Focused default with link" theme="light" level={4} />
```

Comment:
It works: 
![image](https://user-images.githubusercontent.com/35959222/152522285-2f2f42da-6fe9-4543-a48b-50a5b149e8b3.png)

Reply:
If the only way to make the focus is using the tab function, this case should be in a different story (inside a play function).

Reply:
Why do we need to separate this case?

Reply:
Because this test needs an interaction, so it needs to be inside a play function. But only this case and not the rest of them as they don't need interactions.